### DISCUSSION TITLE
Asus GL702VSK fan speed limit

### DISCUSSION THREAD

**Discussion by bonezinger** [dominiksalvet/asus-fan-control#88] [Q&A]
First, thanks a ton for this utility, i've been dealing with a lot less overheating!

Now, I have a GL702VSK (ROG) and the max speed the fan will hit is 6k rpm. This also happens under windows unless you use the Asus ROG Key program, which has a fanboost. If you set it to max, it goes way over 6k, though it doesn't show how high it actually goes.

I'm on Linux Mint and looking at the fan speed with `xsensors`, it looks like it can reach up to 10k rpm, since it shows a small progress bar and 5k is about the middle of it. Where should I look into for trying to "unlock" the actual max speed the fan can reach?

As a side note, `pwmconfig` and `lm-sensors` don't detect the fans.

**Comment by dominiksalvet** [comment]
Hello @bonezinger and welcome to the AFC community! 😃

I am really happy that this utility has made your life a bit easier! 👍I have noticed, however, that you have GL702VSK that I do not see being tested. Please let us add this device as tested if it works and then we can continue with this issue investigation.

Please have a look at the contributing.md file and perform steps described there for adding your device.

**Comment by dominiksalvet** [comment]
So to the 6K issue, if the 6K/10K switch is located somewhere in the ACPI table (which I believe it is), it will be pretty easy to control. It would be able to reuse AFC ACPI internals to perform a valid ACPI call.

The problem, however, is to determine which address is responsible for that. I would try to search on the Internet and then check with afc-scout before overwriting it with another value.

Maybe someone has any insight to this issue. Please contribute if you do! 👍
